### ISSUE TITLE
Make the navigation options programmatically findable

### ISSUE THREAD

**Issue by tidoust** [w3c/htmldiff-nav#4]
This issue tracks some accessibility feedback I received about the navigation options. It may be split or addressed into separate issues as needed.

The navigation options are visually findable, but could be improved to make them progammatically findable. Possible improvements include:

* Put the menu in a <nav> container
* Include a sensible header … something like ’Navigate changes’
* Move it to the start of the DOM
* Consider moving it visually to the top of the page - bottom right is easy to miss
* Include some keyboard options to navigate through the changes

Also:
* When clicking prev/next focus needs to be managed so that users can find the change that has been selected.
* Minor: the current and total number of changes needs some clearer labelling. Doesn’t need to be visible but needs to be there.

(These last two would be considered WCAG failures)

Cc @iadawn, @daniel-montalvo
